tests/main/refresh-undo-haning-stop-cmd/task.yaml

```diff
@@ -0,0 +1,37 @@
+summary: Check that undo works even with a hanging stop command
```

Comment:
There is a typo in the test directory name `s/refresh-undo-haning-stop-cmd/refresh-undo-hanging-stop-cmd/`